_books/ion-1-1/src/whats_new.md

```diff
@@ -15,13 +15,35 @@ compact.
 
 ## Backwards compatibility
 
-Ion 1.1 is backwards compatible with Ion 1.0. While their encodings are distinct, they share the same data model--any data that can be produced and read by an application in Ion 1.1 has an equivalent representation in Ion 1.0.
+Ion 1.0 and Ion 1.1 share the same data model.
+Any data that can be represented in Ion 1.0 can also be represented with full fidelity in Ion 1.1 and vice-versa.
+This means that it is always possible to convert data from one version to the other without risk of data loss.
 
-Ion 1.1 is *not* required to preserve Ion 1.0 binary encodings in Ion 1.1 encoding contexts (i.e., the type codes and
-lower-level encodings are not preserved in the new version). The Ion Version Marker (IVM) is used to denote the
-different versions of the syntax. Ion 1.1 does retain text compatibility with Ion 1.0 in that the changes are a strict
-superset of the grammar, however due to the updated system symbol table, symbol IDs referred to using the `$n` syntax
-for symbols beyond the 1.0 system symbol table are not compatible.
+Ion 1.1 readers must be able to understand both Ion 1.0 and Ion 1.1 data.
+
+The text encoding grammar of Ion 1.1 is a superset of Ion 1.0's text encoding grammar.
+Any Ion 1.0 text data can also be parsed by an Ion 1.1 text parser.
+However, because Ion 1.1 has a different system symbol table,
+symbol IDs in an Ion 1.0 stream do not always refer to the same text as the same symbol ID in an Ion 1.1 stream.
+(For example: in an Ion 1.0 stream, `$4` is always the text `"name"`. However, it may or may not be `"name"` in an Ion 1.1 stream.)
+
+Ion 1.1's binary encoding is substantially different from Ion 1.0's binary encoding.
+Many changes have been made to make values more compact, faster to read and/or faster to write.
+Ion 1.0's [type descriptors](https://amazon-ion.github.io/ion-docs/docs/binary.html#typed-value-formats)
+have been supplanted by Ion 1.1's more general
+[opcodes](https://amazon-ion.github.io/ion-docs/docs/binary.html#typed-value-formats),
+which have been organized to prioritize the most commonly used encodings and make leveraging macros as inexpensive as possible.
+
+In both text and binary Ion 1.1, the Ion Version Marker syntax is syntactically compatible with Ion 1.0's version marker syntax.
+This means that an Ion 1.0-only reader can correctly identify when a stream uses Ion 1.1 (allowing it to report an error),
+and an Ion 1.1 reader can correctly "downshift" to expecting Ion 1.0 data when it encounters a stream using Ion 1.0.
+
+Two streams using different Ion versions can be safely concatenated together provided that they are both text or both binary.
+A concatenated stream containing both Ion 1.0 and Ion 1.1 can only be fully read by a reader that supports Ion 1.1.
+
+Upgrading an existing application to Ion 1.1 often requires little-to-no code changes,
```

Comment:
We should probably caveat this, as using the most impactful feature of Ion 1.1 does require code changes.